lib/backend/k8s/resources/globalnetworkpolicies_test.go

```diff
@@ -118,13 +118,17 @@ var _ = Describe("Global Network Policies conversion methods", func() {
 			kvp1.Value.(*apiv2.GlobalNetworkPolicy).Spec.PreDNAT = true
 			res1.Spec.DoNotTrack = false
 			res1.Spec.PreDNAT = true
+			//TODO: Add back
```

Comment:
Something for @neiljerram to track as part of https://github.com/projectcalico/libcalico-go/pull/580/files?

Reply:
@song-jiang If you haven't already, you should make sure that these lines are eventually uncommented, in your ApplyOnForward work.

Reply:
I did notice that they were uncommented in @song-jiang's ApplyOnForward PR.